app/templates/buyers/review_brief.html

```diff
@@ -0,0 +1,91 @@
+{% extends "_base_page.html" %}
+{% from "components/tabs/macro.njk" import govukTabs %}
+
+{% block pageTitle %}{{ brief.title }} [DRAFT] - Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with
+    items = [
+      {
+          "link": url_for('external.index'),
+          "label": "Digital Marketplace"
+      },
+      {
+          "link": url_for('external.list_opportunities', framework_family=brief.framework.family),
+          "label": "Supplier opportunities"
+      },
+      breadcrumbs
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+
+{% block main_content %}
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <header class="page-heading-smaller">
+      <h1>Review your requirements</h1>
+    </header>
+  </div>
+</div>
```

Comment:
I think we could remove most of this and use only `h1` with govuk-frontend heading class to style the `h1`. As there is nothing else in the heading and it's short so we do not have to add all the extra markup for the grid.

Reply:
I can't get the spacing to work for this without also adding in the `_page_headings.scss` override? I think it might have to wait for now...